api/registrations/serializers.py

```diff
@@ -631,11 +631,9 @@ def _is_attached_file_valid(self, file_metadata, registering):
         Validation of file information on registration_metadata.  Theoretically, the file information
         on registration_responses does not have to be valid, so we enforce their accuracy here,
         to ensure file links load properly.
-
         Verifying that nodeId in the file_metadata is one of the files we're registering. Verify
         that selectedFileName is the name of a file on the node.  Verify that the sha256 matches
         a version on that file.
-
```

Comment:
Add back - removed in schemablocks merge

Reply:
🆗